public/less/_repository.less

```diff
@@ -602,6 +602,9 @@
 		    padding: 6px 40px 4px 35px;
 			}
 		}
+		&.ui.basic.striped.table tbody tr:nth-child(2n) {
```

Comment:
@Unknwon please see reasoning in 70a1cbaf90aa6b13b163d07aeb962858e2bd4903 message, the problem is that grey labels mix with grey strips background, that's why. Alternatively I'd remove the strips altogether.